DistroLauncher/ProcessRunner.cpp

```diff
@@ -17,9 +17,11 @@
 
 #include "stdafx.h"
 
-namespace Helpers {
+namespace Helpers
```

Comment:
I could not grasp what is the relationship of this class with the current PR. If this is just reformattings we should deal with it in another PR.

Reply:
Yeah, I was playing with pre-commit hooks earlier and didn't clean up afterwards. Reverting this.